doc/manual/config-basic.rst

```diff
@@ -107,6 +107,149 @@ Problems can have special *compare* and
 *run* scripts associated to them, to deal with problem
 statements that require non-standard evaluation.
 
+Adding contest data programmatically
+------------------------------------
+
+DOMjudge offers API endpoints to add or update contest data programmatically.
+In general, we try to follow the `CLICS CCS specification
+<https://clics.ecs.baylor.edu/index.php?title=Contest_Control_System_Requirements>`_
+for all file formats involved.
+
+All of the following examples require you to set up admin credentials in your
+`.netrc <https://www.gnu.org/software/inetutils/manual/html_node/The-_002enetrc-file.html>`_ file.
+You need to install `httpie <https://httpie.org/>`_ and replace the
+``<API_URL>`` in the examples below with the API URL of your local DOMjudge
+installation.
+
+Importing team categories
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Prepare a file called ``groups.tsv`` which contains the team categories.
+The first line should contain ``File_Version 1``.
```

Comment:
Perhaps make clear that this also contains a tab

Reply:
Done.